x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/index.tsx

```diff
@@ -84,13 +85,21 @@ export const EndpointDetailsFlyout = memo(() => {
       size="s"
     >
       <EuiFlyoutHeader hasBorder>
-        <EuiTitle size="s">
-          {loading ? (
-            <EuiLoadingContent lines={1} />
-          ) : (
-            <h2 data-test-subj="endpointDetailsFlyoutTitle"> {details?.host?.hostname} </h2>
-          )}
-        </EuiTitle>
+        {loading ? (
+          <EuiLoadingContent lines={1} />
+        ) : (
+          <EuiToolTip content={details?.host?.hostname} anchorClassName="eui-textTruncate">
+            <EuiTitle size="s">
+              <h2
+                style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}
+                data-test-subj="endpointDetailsFlyoutTitle"
+              >
+                {' '}
```

Comment:
Do you need the pre/post spaces?

Reply:
odd, I don't remember adding these, maybe the linter?  Will double check

Reply:
the spaces are now removed, good catch.  They don't seem to affect the solution